setup.cfg

```diff
@@ -17,7 +17,8 @@ packages = find:
 install_requires =
     ansible-core
     ansible-builder>=1.0.1,<2.0
-    ansible-lint>=5.0.8,<6.0
+    ansible-lint>=5.4.0,<6.0; python_version<'3.8'
+    ansible-lint>=6.0.0,<7.0; python_version>='3.8'
```

Comment:
@ssbarnea I think we should keep it so galaxy-importer supports one range of ansible-lint